target/linux/ramips/image/mt76x8.mk

```diff
@@ -855,6 +855,22 @@ define Device/tplink_tl-wr902ac-v3
 endef
 TARGET_DEVICES += tplink_tl-wr902ac-v3
 
+define Device/tplink_tl-wr902ac-v4
+  $(Device/tplink-v2)
+  IMAGE_SIZE := 7808k
+  DEVICE_MODEL := TL-WR902AC
+  DEVICE_VARIANT := v4
+  TPLINK_FLASHLAYOUT := 8Mmtk
+  TPLINK_HWID := 0x000dc88f
+  TPLINK_HWREV := 0x89
+  TPLINK_HWREVADD := 0x1
+  DEVICE_PACKAGES := kmod-mt7615e kmod-mt7663-firmware-ap kmod-usb2 kmod-usb-ohci \
+	kmod-usb-ledtrig-usbport
```

Comment:
Please show me the shell output of `dmesg |grep -i mt`.

Reply:
> Please show me the shell output of `dmesg |grep -i mt`.

```
[    0.000000] SoC Type: MediaTek MT7628AN ver:1 eco:2
[    0.698106] mt7621_gpio 10000600.gpio: registering 32 gpios
[    0.703972] mt7621_gpio 10000600.gpio: registering 32 gpios
[    0.709574] mt7621_gpio 10000600.gpio: registering 32 gpios
[    0.755441] spi-mt7621 10000b00.spi: sys_freq: 193333333
[    0.781925] 4 fixed-partitions partitions found on MTD device spi0.0
[    0.803711] Creating 4 MTD partitions on "spi0.0":
[    0.824029] 2 tplink-fw partitions found on MTD device firmware
[    0.830070] Creating 2 MTD partitions on "firmware":
[    0.840187] mtd: partition "kernel" doesn't end on an erase/write block -- force read-only
[    0.856876] mtd: partition "rootfs" doesn't start on an erase/write block boundary -- force read-only
[    0.867506] mtd: setting mtd3 (rootfs) as root device
[    0.873874] 1 squashfs-split partitions found on MTD device rootfs
[    0.970387] mtk_soc_eth 10100000.ethernet eth0: mediatek frame engine at 0xb0100000, irq 5
[   14.924539] mt76_wmac 10300000.wmac: ASIC revision: 76280001
[   16.034740] mt76_wmac 10300000.wmac: Firmware Version: 20151201
[   16.040784] mt76_wmac 10300000.wmac: Build Time: 20151201183641
[   16.093434] mt76_wmac 10300000.wmac: firmware init done
[   16.315698] mt7615e 0000:01:00.0: card - bus=0x1, slot = 0x0 irq=4
[   17.804773] mt7615e 0000:01:00.0: mediatek/mt7663pr2h.bin not found, switching to mediatek/mt7663pr2h_rebb.bin
[   18.383466] mt7615e 0000:01:00.0: HW/SW Version: 0x65322d31, Build Time: 2009041715da1a1
[   18.893895] mt7615e 0000:01:00.0: N9 Firmware Version: 7663mp1827, Build Time: 20200904171623
[   18.902581] mt7615e 0000:01:00.0: Region number: 0x3
[   18.907644] mt7615e 0000:01:00.0: Parsing tailer Region: 0
[   18.966419] mt7615e 0000:01:00.0: Region 0, override_addr = 0x00112c00
[   18.973087] mt7615e 0000:01:00.0: Parsing tailer Region: 1
[   19.029104] mt7615e 0000:01:00.0: Parsing tailer Region: 2
[   19.063649] mt7615e 0000:01:00.0: override_addr = 0x00112c00, option = 3
[   37.471987] jffs2_build_filesystem(): unlocking the mtd device... 
```